packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -2,13 +2,15 @@
 <!--BEGIN STABILITY BANNER-->
 ---
 
-![cfn-resources: Stable](https://img.shields.io/badge/cfn--resources-stable-success.svg?style=for-the-badge)
+| Features | Stability |
+| --- | --- |
+| CFN Resources | Stable |
```

Comment:
Can we make these into sexy badges? And put the explanations here as well?

Reply:
What kind of explanations should I put in the table? 
I played around with that but nothing clean, mainly because what's under 'stability' is technically self-explanatory; and the next best thing are the paragraphs written underneath.